libcrux-ml-dsa/tests/wycheproof_verify.rs

```diff
@@ -25,7 +25,7 @@ macro_rules! wycheproof_verify_test {
                     assert_eq!(test_group.tests.len(), 1);
                     assert!(test_group.tests[0]
                         .flags
-                        .contains(&Flag::IncorrectPublicKeyLength));
+                        .contains(&"IncorrectPublicKeyLength".into()));
```

Comment:
Hm, what we had here before looked nicer. What's the reason for this change? Now we have to know the exact string here.

Reply:
Yeah I think keeping the enum and just updating when we get new variants sounds better to me.